recipes/metomi-isodatetime/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "metomi-isodatetime" %}
+{% set version = "1!2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a8dd8a12f4df79098c297de192d6dd75b93c78a2565f10ec13281f2f2c5faf29
+
+build:
+  skip: True  # [not linux]
```

This looks like a general Python package without any binary code, so the conda package is neither specific to the OS nor the Python version and can therefore be `noarch: python`
```suggestion
  noarch: python
```

---

I saw this in the example recipe, but wasn't sure if this would work. Thanks @xhochy !